frontend/src/components/MapView/Layers/LayerDropdown.tsx

```diff
@@ -67,6 +67,16 @@ function LayerDropdown({
     tables: [],
   };
 
+  const filterLayersForAnalysis = (layer: LayerType) => {
+    if (layer.type === 'wms') {
+      return (
+        !layer.disableAnalysis &&
```

Comment:
By design, I think layer.disableAnalysis should affect all layers, not just wms layers.